client/src/components/Lesson/WordCreator/ContextSentences.tsx

```diff
@@ -26,6 +29,10 @@ export class ContextSentences extends React.Component<ContextSentencesProps, Con
 
     constructor(props: ContextSentencesProps) {
         super(props);
+        this.state = {
+            value: ([] as string[]),
+            error: ([] as string[]),
```

Comment:
Just noticed that we have the type defined above in the interface so I wouldn't include the type here (`error: []`).

We should still do `const result: number[] = [];` below though